types/events.ts

```diff
@@ -0,0 +1,184 @@
+    attributes: {
+        createdAt: string
+    }
+    relationships: {
+        application: Relationship
+    }
+}
+
+interface AuthorizationCreated {
+    id: string
+    type: "authorization.created"
+    attributes: {
+        createdAt: string
+        cardLast4Digits: string
+        recurring: boolean
+    }
+    relationships: {
+        authorization: Relationship
+        account: Relationship
+        customer: Relationship
+    }
+}
+
+interface CardActivated {
+    id: string
+    type: "card.activated"
+    attributes: {
+        createdAt: string
+    }
+    relationships: {
+        card: Relationship
+        account: Relationship
+        customer: Relationship
+    }
+}
+
+interface CardStatusChanged {
+    id: string
+    type: "card.statusChanged"
+    attributes: {
+        createdAt: string
+        newStatus: string
+        previousStatus: string
+    }
+    relationships: {
+        card: Relationship
+        account: Relationship
+        customer: Relationship
+    }
+}
+
+interface CustomerCreated {
+    id: string
+    type: "customer.created"
+    attributes: {
+        createdAt: string
+    }
+    relationships: {
+        customer: Relationship
+        application: Relationship
+    }
+}
+
+interface DocumentApproved {
+    id: string
+    type: "document.approved"
+    attributes: {
+        createdAt: string
+    }
+    relationships: {
+        document: Relationship
+        application: Relationship
+    }
+}
+
+interface DocumentRejected {
+    id: string
+    type: "document.rejected"
+    attributes: {
+        createdAt: string
+        reason: string
+        reasonCode: string
+    }
+    relationships: {
+        document: Relationship
+        application: Relationship
+    }
+}
+
+interface PaymentClearing {
+    id: string
+    type: "payment.clearing"
+    attributes: {
+        createdAt: string
+        previousStatus: string
+    }
+    relationships: {
+        payment: Relationship
+        data: Relationship
+        customer: Relationship
+    }
+}
+
+interface PaymentSent {
+    id: string
+    type: "payment.sent"
+    attributes: {
+        createdAt: string
+        previousStatus: string
+    }
+    relationships: {
+        payment: Relationship
+        data: Relationship
+        customer: Relationship
+    }
+}
+
+interface PaymentReturned {
+    id: string
+    type: "payment.returned"
+    attributes: {
+        createdAt: string
+        previousStatus: string
+    }
+    relationships: {
+        payment: Relationship
+        data: Relationship
+        customer: Relationship
+    }
+}
+
+interface StatementsCreated {
+    id: string
+    type: "statements.created"
+    attributes: {
+        createdAt: string
+    }
+}
+
+interface TransactionCreated {
+    id: string
+    type: "transaction.created"
+    attributes: {
+        createdAt: string
+        summary: string
+        direction: string
+        amount: string
+    }
+}
```

Comment:
Missing relationships? https://developers.unit.co/#transactioncreated

Reply:
when you are right you are right, thanks !